src/SwedbankPay/Api/Client/ClientVersion.php

```diff
@@ -0,0 +1,185 @@
+      * @return bool true if successful; otherwise false.
+      */
+    private function tryGetVersionNumberFromEnv(&$version) : bool
+    {
+        if (getenv("VERSION") !== false) {
+            $version = getenv("VERSION");
+            return true;
+        }
+
+        return false;
+    }
+
+
+    /**
+      * Tries to get the version number from the composer.json file.
+      * Returns true if successful; otherwise false.
+      *
+      * @param string $version The by-reference $version variable to assign the version number to, if found.
+      * @return bool true if successful; otherwise false.
+      */
+    private function tryGetVersionNumberFromComposerJson(&$version) : bool
+    {
+        $composerPath = __DIR__ . '/../../../../composer.json';
+        $composer = null;
+
+        if (!$this->tryReadJson($composerPath, $composer)) {
+            return false;
+        }
+
+        if (isset($composer['version'])) {
+            $version = $composer['version'];
+            return true;
+        }
+
+        return false;
+    }
+
+
+    /**
+      * Tries to get the version number from the composer.lock file.
+      * Returns true if successful; otherwise false.
+      *
+      * @param string $version The by-reference $version variable to assign the version number to, if found.
+      * @return bool true if successful; otherwise false.
+      */
+    private function tryGetVersionNumberFromComposerLock(&$version) : bool
+    {
+        $composerLockPath = getcwd() . DIRECTORY_SEPARATOR . 'composer.lock';
+        $composerLock = null;
+
+        if (!$this->tryReadJson($composerLockPath, $composerLock)) {
+            return false;
+        }
+
+        if (isset($composerLock['packages'])) {
+            $packages = $composerLock['packages'];
+            foreach ($packages as $package) {
+                if (!isset($package['name']) ||
+                    $package['name'] != "swedbank-pay/swedbank-pay-sdk-php") {
+                    continue;
+                }
+
+                if (isset($package['version'])) {
+                    $version = $package['version'];
+                    return true;
+                }
+            }
+        }
+
+        return false;
+    }
+
+
+    /**
+     * Tries to read the file at $path and assigns a JSON decoded object to
+     * $json if successful. Returns true if successful; otherwise false.
+     *
+     * @param string $path The path of the file to read and JSON decode.
+     * @param string $json The by-reference $json variable to assign the JSON decoded object to.
+     * @return bool true if the JSON decoding is successful; otherwise false.
+     */
+    private function tryReadJson($path, &$json) : bool
+    {
+        $pathResolver = new PathResolver();
+        $path = $pathResolver->resolve($path);
+
+        if (!file_exists($path)) {
+            return false;
+        }
+
+        $contents = file_get_contents($path);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function file_get_contents() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5312212)